cmd/cmd.go

```diff
@@ -741,13 +743,11 @@ func twoLines(s string) [][]string {
 	lines := strings.Split(s, "\n")
 	res := [][]string{}
 
-	count := 0
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
 		if line != "" {
-			count++
 			res = append(res, []string{line})
-			if count == 2 {
+			if len(res) == 2 {
```

Comment:
len(res) is just accessing size of slice, and hence has same compute/memory cost.